src/Microsoft.AspNet.Mvc.Razor.Host/MvcRazorHost.cs

```diff
@@ -95,7 +95,7 @@ internal MvcRazorHost(IChunkTreeCache chunkTreeCache, RazorPathNormalizer pathNo
                     ScopeManagerBeginMethodName = nameof(TagHelperScopeManager.Begin),
                     ScopeManagerEndMethodName = nameof(TagHelperScopeManager.End),
 
-                    TagHelperContentTypeName = nameof(TagHelperContent),
+                    TagHelperContentTypeName = typeof(TagHelperContent).FullName,
```

Comment:
Super solid :+1: will make the change. It's going to be a bit broader of a change with that modification.